modules/ui/fields/wikipedia.js

```diff
@@ -306,6 +307,15 @@ export function uiFieldWikipedia(field, context) {
     }
   }
 
+  function updateEncodedWikiUrl(tagLang, tagArticleTitle, anchor) {
+    const underscoredUrlEncodedTitle = encodeURIComponent(tagArticleTitle.replace(/ /g, '_'));
+    const urlAnchored = toUrlAnchor(anchor);
+    _wikiURL = `${scheme}${tagLang}.${domain}/wiki/${underscoredUrlEncodedTitle}${urlAnchored}`;
```

Comment:
Perhaps this would be more testable if it returns the URL instead of setting `_wikiURL`.

Reply:
Good idea!

Reply:
Made the function return, and assign to wiki so that the tests can invoke it.